### PR TITLE
Add Merqury

### LINKED ISSUE #1614: new module: Merqury
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by mahesh-panchal**
Adds the tool merqury for plotting assembly k-mer spectra.

## PR checklist

Closes #1614  

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by mahesh-panchal** [comment]
ggplot missing font dependency is causing mismatched md5sums between docker and conda.

**Comment by mahesh-panchal** [comment]
Fixed font in container https://github.com/bioconda/bioconda-recipes/pull/35701

File: modules/merqury/meta.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  - "k-mer"
+  - "assembly"
+  - "evaluation"
+tools:
+  - "merqury":
+      description: "Evaluate genome assemblies with k-mers and more."
+      homepage: "None"
+      documentation: "None"
+      tool_dev_url: "https://github.com/marbl/merqury"
+      doi: ""
```
Comment (d4straub) [inline modules/merqury/meta.yml:13]:
```suggestion
      doi: "10.1186/s13059-020-02134-9"
```
as in e.g. https://github.com/nf-core/modules/blob/d8bef6057bc2cf657d33ed496426be8416edd780/modules/antismash/antismashlite/meta.yml#L25?

**Review by d4straub — COMMENTED** [review]
Looks good to me except that potential cpu usage.

File: modules/merqury/meta.yml (hunk 1 above)
Comment (mahesh-panchal) [reply modules/merqury/meta.yml]: I always forget the metadata. Thanks.

File: tests/modules/merqury/test.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  command: nextflow run ./tests/modules/merqury -entry test_merqury -c ./tests/config/nextflow.config  -c ./tests/modules/merqury/nextflow.config
+  tags:
+    - merqury
+  files:
+    - path: output/merqury/genome_only.bed
+      md5sum: b611f22cde0e410a2ca07c1eefd042d3
+    - path: output/merqury/genome_only.wig
+      md5sum: 19cf44989af72af597ef80d3489b4882
+    - path: output/merqury/test.completeness.stats
+      md5sum: d41d8cd98f00b204e9800998ecf8427e
```
Comment (d4straub) [inline tests/modules/merqury/test.yml:11]: That seems the offending empty file md5sum? I do not know how that works now, but maybe removing the checksum and check only for file existence would solve the linting?

**Review by d4straub — APPROVED** [review]
LGTM!

**Pull request merged**